File: BibliotecaDeCodigoPMI/src/main/java/com/example/bibliotecadecodigopmi/Tarea.java
package com.example.bibliotecadecodigopmi;
import java.time.LocalDate;
import java.util.List;
public class Tarea {
    private String Nombre;
    private LocalDate FechaDeInicio;
    private LocalDate FechaDeTerminado;
    private String Descripcion;
    private List<Recurso> recursos;
    public Tarea(String Nombre, LocalDate FechaDeInicio, LocalDate FechaDeTerminado, String Descripcion){
        this.Nombre = Nombre;
        this.FechaDeInicio = FechaDeInicio;
        this.FechaDeTerminado = FechaDeTerminado;
        this.Descripcion= Descripcion;
    }
    public String getNombre() {
        return Nombre;
    }
    public LocalDate getFechaDeInicio() {
        return FechaDeInicio;
    }
    public LocalDate getFechaDeTerminado() {
        return FechaDeTerminado;
    }
    public void setResources(Recurso recurso) {
        this.recursos.add(recurso);
    }
    public List<Recurso> getResources() {
        return recursos;
    }

    @Override
    public String toString() {
        return Nombre;
    }

    public String getDescripcion() {
        return Descripcion;
    }

    public void setNombre(String text) {
        this.Nombre = text;
    }

    public void setFechaDeInicio(LocalDate value) {
        this.FechaDeInicio = value;
    }

    public void setFechaDeTerminado(LocalDate value) {
        this.FechaDeInicio = value;
    }

    public void setDescripcion(String text) {
        this.Descripcion = text;
    }
}

File: BibliotecaDeCodigoPython/com/example/bibliotecadecodigopmi/Tarea.py
#!/usr/bin/python
# -*- coding: UTF-8 -*-
from com.example.bibliotecadecodigopmi import Recurso

class Tarea(object):
	def Tarea(self, aNombre, aFechaDeInicio, aFechaDeTerminado, aDescripcion):
		"""@ParamType aNombre String
		@ParamType aFechaDeInicio java.time.LocalDate
		@ParamType aFechaDeTerminado java.time.LocalDate
		@ParamType aDescripcion String"""
		pass

	def setResources(self, aRecurso):
		"""@ParamType aRecurso com.example.bibliotecadecodigopmi.Recurso
		@ReturnType void"""
		pass

	def getResources(self):
		"""@ReturnType java.util.List"""
		pass

	def toString(self):
		"""@ReturnType String"""
		pass

	def setNombre(self, aNombre):
		""""""@ParamType aNombre String"""
		@ParamType aNombre String"""
		self.___nombre = aNombre

	def getNombre(self):
		""""""@ReturnType String"""
		@ReturnType String"""
		return self.___nombre

	def setFechaDeInicio(self, aFechaDeInicio):
		""""""@ParamType aFechaDeInicio java.time.LocalDate"""
		@ParamType aFechaDeInicio java.time.LocalDate"""
		self.___fechaDeInicio = aFechaDeInicio

	def getFechaDeInicio(self):
		""""""@ReturnType java.time.LocalDate"""
		@ReturnType java.time.LocalDate"""
		return self.___fechaDeInicio

	def setFechaDeTerminado(self, aFechaDeTerminado):
		""""""@ParamType aFechaDeTerminado java.time.LocalDate"""
		@ParamType aFechaDeTerminado java.time.LocalDate"""
		self.___fechaDeTerminado = aFechaDeTerminado

	def getFechaDeTerminado(self):
		""""""@ReturnType java.time.LocalDate"""
		@ReturnType java.time.LocalDate"""
		return self.___fechaDeTerminado

	def setDescripcion(self, aDescripcion):
		""""""@ParamType aDescripcion String"""
		@ParamType aDescripcion String"""
		self.___descripcion = aDescripcion

	def getDescripcion(self):
		""""""@ReturnType String"""
		@ReturnType String"""
		return self.___descripcion

	def __init__(self):
		self.___nombre = None
		"""@AttributeType String"""
		self.___fechaDeInicio = None
		"""@AttributeType java.time.LocalDate"""
		self.___fechaDeTerminado = None
		"""@AttributeType java.time.LocalDate"""
		self.___descripcion = None
		"""@AttributeType String"""
		self.___recursos = None
		# @AssociationType com.example.bibliotecadecodigopmi.Recurso*
		# @AssociationMultiplicity *

	Nombre = property(getNombre, setNombre)

	FechaDeInicio = property(getFechaDeInicio, setFechaDeInicio)

	FechaDeTerminado = property(getFechaDeTerminado, setFechaDeTerminado)

	Descripcion = property(getDescripcion, setDescripcion)

